tests/src/benches.rs

```diff
@@ -91,7 +92,7 @@ impl BenchWorld {
         let book = FontBook::from_fonts([&font]);
 
         Self {
-            library: Prehashed::new(Library::build()),
+            library: Prehashed::new(Library::build(SysArguments::default())),
```

Comment:
When seeing this, I'm unsure whether `SysArguments` is really that great. Maybe it should rather be a `LibraryBuilder` struct and we have `Library::builder() -> LibraryBuilder` and `LibraryBuilder::build() -> Library`. The places for `SysArguments::default()` are used would then just use `Library::default()` and those that require further configuration can use `Library::builder().with_inputs(..).build()`.